doc/modules/model_evaluation.rst

```diff
@@ -70,7 +70,7 @@ Scoring                           Function
 'neg_log_loss'                    :func:`metrics.log_loss`                          requires ``predict_proba`` support
 'precision' etc.                  :func:`metrics.precision_score`                   suffixes apply as with 'f1'
 'recall' etc.                     :func:`metrics.recall_score`                      suffixes apply as with 'f1'
-'jaccard' etc.                    :func:`metric.jaccard_similarity_score`           suffixes apply as with 'f1'
+'jaccard' etc.                    :func:`metrics.jaccard_similarity_score`          suffixes apply as with 'f1'
```

Comment:
You still need to add scorers to `sklearn/metrics/scorer.py` if you reference them here.